README.md

```diff
@@ -45,6 +45,8 @@ make container-serve
 
 Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
 
+If you see errors, it probably means that the hugo container did not have enough computing resources available. To solve it, increase the amount of allowed CPU and memory usage for Docker on your machine ([MacOSX](https://docs.docker.com/docker-for-mac/#resources) and [Windows](https://docs.docker.com/docker-for-windows/#resources)).
```

Comment:
Hi @remyleone .

Does it make sense to add this note immediately after the make "container" instructions?

Reply:
It does :) I've fixed it.